packages/typespec-test/test/schemaRegistry/spec/client.tsp

```diff
@@ -4,13 +4,13 @@ import "./main.tsp";
 using Azure.ClientGenerator.Core;
 
 @@access(Azure.SchemaRegistry.Version, Access.internal);
-@@projectedName(Azure.SchemaRegistry.Version, "client", "SchemaVersion");
+@@clientName(Azure.SchemaRegistry.Version, "SchemaVersion");
```

Comment:
this is not working, the same as https://github.com/Azure/typespec-azure/issues/730